hummingbird/ml/_container.py

```diff
@@ -81,25 +80,105 @@ def _run(self, function, *inputs, reshape=False):
             splits = [inputs[input_names[idx]] for idx in range(len(input_names))]
             inputs = [df.to_numpy().reshape(-1, 1) for df in splits]
 
-        if self._batch_size is None:
-            return function(*inputs)
-        else:
-            return self._run_batch_inference(function, *inputs, reshape=reshape)
+        return function(*inputs)
```

Comment:
Nit: the documentation of this method needs a refresh I think (sorry but I wasn't able to add a comment to the proper line for some reason). `This function either score the full dataset at once or triggers batch inference.` Now it always score the full dataset, right?

Reply:
The doc updated. Also removed `reshape=False` option, as it is not used anymore.